includes/reader-revenue/my-account/style.scss

```diff
@@ -48,13 +39,16 @@
 	}
 
 	.woocommerce-message {
-		padding: 28px;
+		background: color.adjust( wp-colors.$alert-green, $lightness: 42% );
+		border-color: wp-colors.$alert-green;
+		color: inherit;
```

Comment:
These should be applied to the code block at the top, where the `:before` was. Otherwise, the background is not applied to the success message: 

<img width="490" alt="image" src="https://user-images.githubusercontent.com/7383192/187678411-3fad76c6-cf5f-4d84-a9dc-2948acf18fa6.png">

Reply:
Fixed: https://github.com/Automattic/newspack-plugin/pull/1956/commits/3f64830cc4526673fbca12fb1d9a37873544ca1c